### ISSUE TITLE
Expand label prediction and transfer methods

### ISSUE THREAD

**Issue by mumichae** [HCA-integration/scAtlasTb#316]
## Cell type prediction 

Concept: Predicting labels from pre-trained model

- [x] Celltypist
- [ ] scTab
- [ ] SCimilarity

## Label transfer

- [x]  majority voting
- [ ] XGBoost
- [ ] kNN classifier

## Additional methods
* DOT, cell2location --> deconvolution
